src/unity/paths/orgs.yml

```diff
@@ -18,19 +18,19 @@ post:
         application/json:
           schema:
             $ref: '../schemas/OrganizationWithToken.yml'
+    '400':
```

Comment:
So, even if we need the `400`, I think we'll still need the `422` additionally, because `Quartz.provision_organization` can return `{:error, %ValidationError.t()}`, which `FallbackController` turns into a `422`.